charts/console/templates/deployment.yaml

```
@@ -20,6 +20,7 @@ spec:
       {{- end }}
       labels:
         {{- include "console.selectorLabels" . | nindent 8 }}
+        {{- with .Values.podLabels }}{{ toYaml . | nindent 8 }}{{ end }}
```

No need for `with` here.
```suggestion
        {{ toYaml .Values.podLabels | nindent 8 }}
```

---

Fine with me ;)